implemented task and tests (#1)
mate-academy/layout_style-it-up: readme.md

# Style it up

Your task is to change `style.css` in order to get the page in `index.html` 
looking and behaving just as described below.

Please **do not** modify or replace the HTML itself. Do take a look at what’s 
inside `index.html` though.

The page consists of a header, a footer, and 9 blocks between them.

The width of the blocks must be 300 pixels. The height of each of the blocks 
(including both header and footer) has to be 100 pixels. When hovered (the 

> cursor is on the block), it must increase to 200 pixels.

Reviewer: ```suggestion
cursor is on the block), height must increase to 200 pixels.
```

The color of the header must be blue, the color of the footer must be black, and 

> the colors of the blocks in between must be red, then green, then orange, then 

Reviewer: ```suggestion
the colors of the blocks in between must be the following
1. red
1. green
2. orange
... 
```

again red, then green, then orange, and once again red, then green, then orange.

There should be no vertical spacing between the blocks and the edges of the 
page. Place the blocks so that they are horizontally centered on the page (you 

> can use `margin: auto` for that).

Reviewer: I think we are providing too many details. They should figure out themselves how to center the element.

Reviewer 2: Agree with Roman.

# Style it up

Your task is to change `style.css` in order to get the page in `index.html` 
looking and behaving just as described below.

Please **do not** modify or replace the HTML itself. Do take a look at what’s 
inside `index.html` though.

The page consists of a header, a footer, and 9 blocks between them.

The width of the blocks must be 300 pixels. The height of each of the blocks 
(including both header and footer) has to be 100 pixels. When hovered (the 
cursor is on the block), height must increase to 200 pixels.
The color of the header must be blue, the color of the footer must be black, and 
the colors of the blocks in between must be the following
1. red
1. green

> 2. orange

Reviewer: ```suggestion
1. orange
```

... 

> again red, then green, then orange, and once again red, then green, then orange.

Reviewer: This can be removed now. And let's remove `...` above.
